app/src/main/res/layout/article_view.xml

```diff
@@ -0,0 +1,36 @@
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+
+<LinearLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="horizontal">
```

Comment:
For this, you will probably want to mark the orientation as `vertical`, this should improve some of the readability on this custom view.  You could also apply an `android:layout_margin` or `android:layout_marginBottom` to provide some spacing between each ArticleView.